test/ntp_client_test.rb

```diff
@@ -442,6 +442,17 @@
           expect(subject.sntp_test(server)).to eql(false)
         end
 
+        it "returns false if sntp respong no UCST" do
```

Comment:
just a typo and maybe reword to `it "returns false if sntp response includes 'no UCST'"`